### DISCUSSION TITLE
alist上传文件、复制文件，是否可以跳过已存在的文件？

### DISCUSSION THREAD

**Discussion by gdtv** [AlistGo/alist#7013] [Q&A]
我有大量文件要上传，或者从一个存储器复制到另一个存储器，大概有几十万个文件，好几TB大小。
每次从电脑上传文件，或者在存储器之间复制文件，这需要很长的时候，中途可能出错，可能会遇到要重启alist甚至重启服务器的情况。
现在因为不能跳过已存在的文件，每次都要重新开始，要重新上传/复制已存在的文件，这让我的任务几乎变成了不可能。

**Comment by pongfcnkl** [comment]
用rclone可以跳过重复文件

**Comment by LINJIAXIN-China** [comment]
我也有此需求，主要是多个文件夹同时复制时，看起来复制文件顺序没有规律，不过复制文件前都会先把文件夹给创建了，也没法手动去将遗漏的文件去补充，建议开发者可以通过配置文件来设定重复文件名是否跳过或覆盖的功能
